docs/source/workflow_inputs.rst

```diff
@@ -158,6 +158,7 @@ They can be smooth schedules, or they can reflect real-world or stochastic occup
 Detailed schedule inputs are provided via CSV file(s) that should be referenced in the HPXML file as a ``/HPXML/SoftwareInfo/extension/SchedulesFilePath``.
 
 Occupancy related schedule columns must be normalized to MAX=1; that is, these schedules only define *when* energy is used, not *how much* energy is used.
+Thermostat setpoint schedule columns do not have these constraints.
```

Comment:
Thinking ahead to the WH GEB changes, I suggest creating a new column below called 'Units', which is either `frac` or `F`, and then making this change:

```suggestion
Columns with units of `frac` must be normalized to MAX=1; that is, these schedules only define *when* energy is used, not *how much* energy is used.
```